File: Section Reports/Layout/Charting/C#/Charting/rpt2DBar.cs
using System.Drawing;
namespace GrapeCity.ActiveReports.Samples.Charting
{
	/// <summary>
	/// Description of rpt2DBar.
	/// 
	/// </summary>
	public partial class rpt2DBar : GrapeCity.ActiveReports.SectionReport
	{
		public rpt2DBar()
		{
			//
			// Required for ActiveReport designer support.
			//
			InitializeComponent();
			
			Data.OleDBDataSource dS = new Data.OleDBDataSource();
			dS.ConnectionString = Properties.Resources.ConnectionString;			
			dS.SQL = "SELECT ShipCountry, SUM(Freight) AS Expr1 FROM Orders GROUP BY ShipCountry";
			ChartControl.DataSource = dS;
		}
	}
}

File: Section Reports/Layout/Charting/VB.NET/Charting/rpt2DBar.vb
Public Class rpt2DBar
	Inherits GrapeCity.ActiveReports.SectionReport
	Public Sub New()
		MyBase.New()
		InitializeComponent()
		Dim ds As New Data.OleDBDataSource()
		ds.ConnectionString = My.Resources.ConnectionString
		
		ds.SQL = "SELECT ShipCountry, SUM(Freight) AS Expr1 FROM Orders GROUP BY ShipCountry"
		ChartControl.DataSource = ds
		'
		'
		'
		'
		'
		'
	End Sub
End Class
